dts/bindings/can/microchip,mcp2515.yaml

```diff
@@ -0,0 +1,30 @@
+#
+# Copyright (c) 2019 Karsten Koenig
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+---
+title: MCP2515 CAN
+version: 0.1
+
+description: >
+    This binding gives a base representation of the MCP2515 SPI CAN controller
+
+inherits:
+    !include [can.yaml, spi-device.yaml]
+
+properties:
+    compatible:
+      type: string
+      category: required
+      description: compatible strings
+      constraint: "microchip,mcp2515"
+    int-gpios:
+      type: compound
+      category: required
+      generation: define, use-prop-name
+    cs-gpios:
```

Comment:
cs-gpios shouldn't be here, its handled at the spi controller level.

Reply:
Hmmm I just added this because @DWagenk couldn't get the sample to work otherwise. 
https://github.com/DWagenk/zephyr/commit/4a759f5c94407d223548a4a75b7b1435a1eb8e9f

Does it have to be DT_MICROCHIP_MCP2515_0_CS_GPIOS_PIN or DT_MICROCHIP_MCP2515_0_CS_GPIO_PIN ? Because https://github.com/zephyrproject-rtos/zephyr/blob/master/drivers/ethernet/eth_enc28j60.c is doing it in singular (GPIO instead of GPIOS) According to the spi.yaml it should be GPIOS as far as I understand

Reply:
@galak I looked into the SPI-drivers sources a little bit. Am I understanding correctly, that all of spi-cs via gpio handling is handled at that level already and there's no need to have a spi-cs-gpio appear anywhere in the mcp2515 driver?
So deleting `cs-gpios` section here again and also removing the `config CAN_MCP2515_GPIO_SPI_CS` option from Kconfig.mcp2515 as well as the `#ifdef CONFIG_CAN_MCP2515_GPIO_SPI_CS` sections from mcp2515.c would be the way to go?
This would also apply to the enc28j60 that @karstenkoenig used as a reference for his implementation, correct?

Reply:
I removed the cs-gpios section in the mcp2515 dts bindings, as the one in spi.yaml should be enough. 
As the enc28j60 driver is still using the CONFIG_*_GPIO_SPI_CS it is left in for now here as well. I can probably #ifdef DT_MICROCHIP_MCP2515_0_CS_GPIOS_PIN as it is an optional field, but without any clear indication which would be the right way I'd rather have it consistent at least :-|